create-rust-app_cli/src/main.rs

```diff
@@ -61,8 +61,7 @@ enum Commands {
             name = "non-interactive mode",
             help = "Configure project through interactive CLI arguments.",
             requires = "database",
-            requires = "backend framework",
-            requires = "plugins"
+            requires = "backend framework"
```

Comment:
Don't require plugins to be specified (allow no plugins)

Reply:
I might also change some things in git.rs soon so that if -c is used and user.name and/or user.email aren't specified, it just prints a warning and exits instead of prompting for a name/email.

... 

Actually yeah, I'll do that right now